product_docs/docs/pgd/5/security/verification.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: Verification
+---
+
+PGD was verified using the following tools and approaches.
+
+### Coverity
```

Comment:
Let's try to find out if this is still accurate. If we don't get confirmation, I suggest removing this part.

Reply:
As per [this Slack discussion](https://edb.slack.com/archives/G01FLQYF0H1/p1698915089898509?thread_ts=1698844185.798169&cid=G01FLQYF0H1), let's remove this chapter for now - we don't run Coverity scans anymore.